### ISSUE TITLE
twisted/test/threading_latency.py is an old cruddy looking thing

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3720]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3720|
|Type|defect|
|Created|2009-03-30 19:47:55Z|
|Branch|https://github.com/twisted/twisted/tree/delete-threading-latency-3720|

This is just lying around.  Nothing ever runs it.  We could delete it.  If there's useful stuff in it we could make it into a real test file.

<details><summary>Searchable metadata</summary>

```
trac-id__3720 3720
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_delete_threading_latency_3720 branches/delete-threading-latency-3720
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1238442475000000 1238442475000000
changetime__1239632368000000 1239632368000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [26662]) Branching to 'delete-threading-latency-3720'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

I just deleted it.  The code is broken in a lot of ways, I couldn't even get it to run without non-trivial fixes.  This clearly hasn't been used in years and years, so deleting it from the repository is no major loss.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [26681]) Merge delete-threading-latency-3720

Author: exarkun
Reviewer: therve
Fixes: [#3720](https://github.com/twisted/twisted/issues/3720)

Remove some old, dead code that has just been lying around broken for many years.
This code was probably trying to benchmark certain threading APIs, but did so by
using pyunit in a way which hasn't worked in a long time.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
